R/tm_g_patient_profile.R

```diff
@@ -62,70 +62,32 @@
 #' @export
 #'
 #' @examples
-#' library(nestcolor)
+#' data <- cdisc_data() |>
+#'   within(library(nestcolor)) |>
+#'   within({
+#'     ADSL <- rADSL
```

Comment:
I know `rADSL` comes from `osprey` that is loaded as this packaged Depends on it. However we put `osprey::` and other package names in front of datasets, so that there is no conflict in the namespace. `rADSL` and similar like `rADLB` exist in a couple other packages, so prefixing a package in here makes it clear what is the source of the data.

This is minor so no need to argue or discuss. This is also what's been made in other packages containing `rADSL` and `rADLB`

Reply:
See discussion [here](https://github.com/insightsengineering/teal.osprey/pull/232#discussion_r1395631626).